cytominer_eval/evaluate.py

```diff
@@ -30,6 +33,7 @@ def evaluate(
     grit_control_perts: List[str] = ["None"],
     grit_replicate_summary_method: str = "mean",
     mp_value_params: dict = {},
+    percentile: float = 0.5,
```

Comment:
```suggestion
    enrichment_percentile: float = 0.5,
```

Just to make sure it's clear that `percentile` is only operational with `operation="enrichment"`

Reply:
Done